packages/containers/Bootstrap/Bootstrap.js

```diff
@@ -61,9 +61,14 @@ class Bootstrap extends Component {
     this.props.initCoresCount();
 
     try {
+      const { shared = {}, logout } = this.props;
       const { data: bootstrap } = await getUserBootstrap();
       const { data: user } = await getUserSelf();
 
+      if (user?.email.includes('anonymous') && !shared?.mp) {
```

Comment:
A very dirty hack, we need to add the todo for fix it later.
I think we need to create namespaces for JWT tokens.

Reply:
Good idea with namespaces in JWT.
Created ticket and added TODO in code.